common/OpTestSystem.py

```diff
@@ -87,15 +87,22 @@ class OpTestSystem(object):
     def __init__(self,
                  bmc=None,
                  host=None,
+                 hmc=None,
                  prompt=None,
                  conf=None,
                  state=OpSystemState.UNKNOWN):
         self.conf = conf
         self.util = conf.util
         self.bmc = self.cv_BMC = bmc
+        self.hmc = self.cv_HMC = hmc
         self.cv_HOST = host
         self.cv_IPMI = bmc.get_ipmi()
         self.rest = self.bmc.get_rest_api()
+        if hmc is not None:
+            self.console = self.bmc.get_host_console()
+        else:
+            # Skipping HMC console
+            self.console = None
```

Comment:
Unclear as to why this is needed ?  Seems both paths end up calling self.bmc.get_host_console() ?